ledger/metrics/src/main/scala/com/daml/metrics/PruningMetrics.scala

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import com.daml.metrics.api.MetricName
+import com.daml.metrics.api.MetricHandle.Factory
+
+class PruningMetrics(prefix: MetricName, factory: Factory) {
+  val prune = factory.meter(prefix :+ "prune")
```

Comment:
I think it's worth capturing why we use a meter in a comment here

Reply:
I'd also like to understand how will users be able to use this meter? E.g., how can they query if pruning is on-going? And which phase we're in?